packages/shared/src/components/buttons/Button.tsx

```diff
@@ -45,7 +62,7 @@ function ButtonComponent<TagName extends AllowedTags>(
     pressed,
     icon,
     rightIcon,
-    buttonSize,
+    buttonSize = 'medium',
```

Comment:
What is the size if this is left empty? Be careful with changing the default value as there are a lot of areas where this component was used.

Reply:
It always was medium, but wasn't explicitly specified in props specified, but it can be figured out from button styles

Reply:
Nice! This should be good to go then 🚢